src/util/authUtils.js

```diff
@@ -83,3 +83,21 @@ export const applyServerState = (server, options) => {
 	server.state('refresh_token', authCookieOptions);
 };
 
+export const assignMemberState = options => (request, reply) => {
+	const memberValue = options.API_HOST.includes('.dev.') ?
+		request.state.MEETUP_MEMBER_DEV :
+		request.state.MEETUP_MEMBER;
+	request.state.MEETUP_MEMBER = memberValue;
+	request.state.MEETUP_MEMBER_DEV = memberValue;
```

Comment:
Gotchya

> One thing that we might want to talk about is whether we should restrict dev to pointing to www.api.dev.meetup.com, the way that Meetup Classic dev works, rather than letting devs set the API_HOST env variable themselves

I think I agree with this notion for two reasons: 1) consistency with Meetup Classic will probably mitigate some confusion otherwise since the two systems would work "the same". 2) fewer possible sources of errors/issues on dev... i.e., if you're on dev you always get the dev endpoints then you don't have to double check separate configurations.